packages/klevu-ui/src/components/klevu-merchandising/klevu-merchandising.tsx

```diff
@@ -116,6 +125,11 @@ export class KlevuMerchandising {
   async connectedCallback() {
     await KlevuInit.ready()
     await this.#fetchData()
+
+    const settings = getKMCSettings()
+    if (this.usePersonalisation === undefined && settings?.klevu_uc_userOptions.enablePersonalisationInCatNav) {
+      this.usePersonalisation = true
```

Comment:
would it be better to have it as 
`this.usePersonlization = settings?.klevu_uc_userOptions.enablePersonalisationInCatNav || false`
so that it always is set with a boolean value true/false.

Reply:
Then we would override the the value what user has set with props. That's why there has to be a `=== undefined` check.

Reply:
i mean like this
```
if (this.usePersonalisation === undefined)
  this.usePersonlization = settings?.klevu_uc_userOptions.enablePersonalisationInCatNav || false
```